components/dialog/overview.md

```diff
@@ -12,15 +12,17 @@ position: 0
 
 The Dialog is a modal popup that brings information to the user. It provides actions through its action buttons to prompt the user for input or to ask for a decision. The component can also contain more complex UI elements that require the attention of the user. The main difference from the Window modal is the functionality for actions and predefined dialogs.
 
-## Basics
+## Creating Blazor Dialog
 
-To add a Telerik Dialog to your Blazor app:
+1. Add the `TelerikDialog` tag to a Razor file.
 
-1. Add the `TelerikDialog` tag.
-1. Set the `Visible` parameter via one-way or two-way binding.
-1. Set a `Title`.
+2. Set the `Visible` parameter to a `bool` object. It supports one-way and two-way binding.
 
-The following example demonstrates how to set up the Dialog with its default configuration.
+3. Set the `Title` property to a `string` object.
+
+4. Define the `DialogContent` inside the `TelerikDialog` tag.
```

Comment:
It's not clear if this is a parameter, a child tag, or can be both.

```suggestion
4. Define the `DialogContent` inside the `TelerikDialog` tag.
```

Reply:
both - maybe "Set the dialog content through the `DialogContent` RenderFragment parameter" will be explanatory